code/game/objects/items/clown_items.dm

```diff
@@ -185,10 +185,11 @@
 	throw_range = 7
 	attack_verb_continuous = list("HONKS")
 	attack_verb_simple = list("HONK")
+	var/soundfile = 'sound/items/bikehorn.ogg'
```

Comment:
Doc-comment.

Reply:
no longer exists and replaced with LoadComponent to fix inheritance stacking the component